R/create.multipanelplot.R

```diff
@@ -2,26 +2,17 @@
 ### This is needed because without it, the returned object is a grob which you must call grid.draw(x) on, this way, we can simply print the object.
 
 print.multipanel <- function(x, ...) {
```

Comment:
I am probably missing something but could we call the function `plot.multipanel` and remove L15?

Reply:
We want both `plot` and `print` methods to be defined though. The previous code just copy-pasted the definition twice so this only defines it once and uses the previous definition to assign to `plot.multipanel`. Perhaps more explicit is to write:

```r
print.multipanel <- plot.multipanel <- function(x, ...) {
```

Reply:
Ah, I see. Yes, your suggested code block makes more sense to me.